pkg/gvk/gvk.go

```diff
@@ -19,12 +20,39 @@ var crdGVR = schema.GroupVersionResource{
 	Resource: "customresourcedefinitions",
 }
 
+type cohabitatingResource struct {
+	resource       string
+	groupResource1 schema.GroupResource
+	groupResource2 schema.GroupResource
+	seen           bool
+}
+
+func newCohabitatingResource(resource, group1, group2 string) *cohabitatingResource {
+	return &cohabitatingResource{
+		resource:       resource,
+		groupResource1: schema.GroupResource{Group: group1, Resource: resource},
+		groupResource2: schema.GroupResource{Group: group2, Resource: resource},
+		seen:           false,
+	}
+}
+
+func NewCohabitatingResources() map[string]*cohabitatingResource {
```

Comment:
Does this need to be exported?

Reply:
Yes, I believe so because it is used in another package for instantiation[[1]](https://github.com/konveyor/mig-controller/pull/534/files#diff-ff4cf8d4f8669d3270d95d4c88c43266R67).

1. https://github.com/konveyor/mig-controller/pull/534/files#diff-ff4cf8d4f8669d3270d95d4c88c43266R67